Monika After Story/game/script-topics.rpy

```diff
@@ -4782,3 +4782,44 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_explain",category=['romance'],prompt="Can you explain our relationship to someone?",random=False))
+    m 1c "You want me to explain our relationship to someone else?"
```

Comment:
You need to have a label here. Refer back to the contributing guide for how this works.